src/__tests__/Utils-spec.js

```diff
@@ -196,4 +200,48 @@ describe('Utils', () => {
 
   });
 
+  describe('selectorType', () => {
+
+    it('returns CLASS_TYPE for a prefixed .', () => {
+      const type = selectorType('.foo');
+
+      expect(type).to.be.equal(SELECTOR.CLASS_TYPE);
+    });
+
+    it('returns ID_TYPE for a prefixed #', () => {
+      const type = selectorType('#foo');
+
+      expect(type).to.be.equal(SELECTOR.ID_TYPE);
+    });
+
+    it('returns PROP_TYPE for []', () => {
+      function isProp(selector) {
+        expect(selectorType(selector)).to.be.equal(SELECTOR.PROP_TYPE);
+      }
+
+      isProp('[foo]');
+      isProp('[foo="bar"]');
+    });
+
+  });
+
+  describe('coercePropValue', () => {
+
+    it('returns undefined if passed undefined', () => {
+      expect(coercePropValue(undefined)).to.be.undefined;
+    });
+
+    it('returns number if passed a stringified number', () => {
+      expect(coercePropValue('1')).to.be.equal(1);
+      expect(coercePropValue('0')).to.be.equal(0);
+    });
+
+    it('returns a boolean if passed a stringified bool', () => {
+      expect(coercePropValue('true')).to.be.true;
+      expect(coercePropValue('false')).to.be.false;
```

Comment:
when I run the linter, those aren't reported. But I changed them, we'll see if this gets it passing